content/rs/references/rest-api/objects/shard/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+Title: Shard object
+linkTitle: shard
+description: An object that represents a database shard
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+
+An API object that represents a Redis shard in a bdb.
```

Comment:
```suggestion
An API object that represents a Redis shard in a database.
```
Is it possible to use database here instead of bdb?

Reply:
Fixed.